packages/neos-ui-redux-store/src/UI/EditModePanel/index.js

```diff
@@ -30,11 +30,9 @@ export const actions = {
 // Export the reducer
 //
 export const reducer = handleActions({
-    [system.INIT]: () => $set(
+    [system.INIT]: payload => $set(
         'ui.editModePanel',
-        new Map({
-            isHidden: true
```

Comment:
we could; although I quite like that you see the state structure in the reducers.